utils/digital_interrupts.go

```diff
@@ -48,15 +47,7 @@ func CreateDigitalInterrupt(cfg DigitalInterruptConfig) (ReconfigurableDigitalIn
 		cfg.Type = "basic"
```

Comment:
This if statement is invalid now that we only have 'basic' type.

Reply:
I think this is a good time to get rid of cfg.Type.

Reply:
No, @oliviamiller's just approved scope doc has a "type" config value for pins; there is a refactor that gets rid of basic interrupt altogether, but I am not folding in that work with the current pr.

Reply:
I'll get rid of the if check though